packages/jellyfish-transaction/src/script/mapping.ts

```diff
@@ -353,7 +353,14 @@ export const OP_CODES = {
       data: createLoanScheme
     })
   },
-
+  OP_DEFI_TX_ICX_SUBMIT_EXT_HTLC: (icxSubmitEXTHTLC: ICXSubmitEXTHTLC): OP_DEFI_TX => {
```

Comment:
CreateLoanScheme should be arranged after this

Reply:
https://github.com/DeFiCh/jellyfish/pull/589/files#r694550426